dashboard/config/routes.rb

```diff
@@ -5,6 +5,7 @@ module OPS
   DASHBOARDAPI = 'dashboardapi' unless defined? DASHBOARDAPI
 end
 
+# rubocop:disable Metrics/BlockLength
```

Comment:
Prefer adding this rule to the exclude list, along with the other `Metrics/*` rules: https://github.com/code-dot-org/code-dot-org/blob/ed5bc8fadf0d7720c4d9b3ddc167843d8ac40224/.rubocop.yml#L47-L61